src/components/app/MenuButtons.js

```diff
@@ -578,6 +578,21 @@ type MenuButtonsProps = ConnectedProps<
   MenuButtonsDispatchProps
 >;
 
+function _mapMetaInfoExtensionNames(data: any) {
+  const extensionData = data;
+  const extensionList = extensionData.map(d => (
+    <li className="metainfoListItem" key={d}>
+      {d}
+    </li>
+  ));
+  return extensionList;
+}
+
+function _formatDate(timestamp: number): string {
+  const timestampDate = new Date(timestamp).toUTCString();
```

Comment:
Oh, and I'm not sure how this will behave in testing environments, we me need to mock out the `navigator.language` value to get consistent test results.

https://jestjs.io/docs/en/jest-object.html#jestspyonobject-methodname-accesstype

Reply:
Filed a follow up #1447